OKR.md

```diff
@@ -0,0 +1,52 @@
+# Quarterly Objectives and Key Results
+
+We try to frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
+
+## 2018 Q4
+
+**go-ipfs handles large datasets (1TB++) without a sweat**
+- `PX` - It takes less than 48 hours to transfer 1TB dataset over Fast Ethernet (100Mbps)
+- `PX` - It takes less than 12 hours to transfer 200GB sharded dataset over Fast Ethernet (100Mbps)
+- `PX` - There is a prototype implementation of GraphSync for go-ipfs
+- `PX` - There is a better and more performant datastore module (e.g Badger or better)
+- `P1` - Rewrite pinning data structures to support large data sets / many files performantly
+
+**The bandwidth usage is reduced significantly and is well kept under control**
+- `PX` - Users can opt out of providing every IPLD node (and only provide root hashes)
+- `PX` - "Bitswap improvements around reducing chattiness, decreasing bandwidth usage (fewer dupe blocks), and increasing throughput"
+
+**It is a joy to use go-ipfs programatically**
+- `PX` @magik6k - The Core API is finalized and released. Make it easier to import go-ipfs as a package
+- `PX` - go-ipfs-api exposes the new Core API
+- `PX` - go-ipfs Daemon uses the new Core API
+- `PX` - go-ipfs Gateway uses the new Core API
+- `PX` - go-ipfs-cms uses uses the new Core API
+- `PX` - The legacy non Core API is deprecated and the diagram on go-ipfs README is updated
+
+**go-ipfs becomes a well maintained project**
+- `PX` - The Go Contributing Guidelines are updated to contemplate expecations from Core Devs and instructions on how to be an effective contributor (e.g include PR templates)
```

Comment:
Could we make this more precise? Contributing Guidelines describe X practices/habits necessary to be an effective core contributor (including x, y, z)